packages/@aws-cdk/aws-lambda/README.md

```diff
@@ -632,6 +632,26 @@ fn.addEventSource(new eventsources.S3EventSource(bucket, {
 }));
 ```
 
+The following code adds an DynamoDB notification as an event source filtering insert events:
+
+```ts
+import * as eventsources from '@aws-cdk/aws-lambda-event-sources';
+import * as dynamodb from '@aws-cdk/aws-dynamodb';
+
+declare const fn: lambda.Function;
+const table = new dynamodb.Table(this, 'Table', {
+  partitionKey: {
+    name: 'id',
+    type: dynamodb.AttributeType.STRING,
+  },
+  stream: dynamodb.StreamViewType.NEW_IMAGE,
+});
+fn.addEventSource(new eventsources.DynamoEventSource(table, {
+  startingPosition: lambda.StartingPosition.LATEST,
+  filters: [{ eventName: lambda.FilterRule.isEqual('INSERT') }],
```

Comment:
So, I misread your intention here. I'm going to take the input to one of your integration tests here to clarify what I thought you meant here:

Instead of this:
```
    new EventSourceMapping(stack, 'test', {
      target: fn,
      eventSourceArn: eventSourceArn,
      kafkaTopic: topicNameParam.valueAsString,
      filters: [{
        orFilter: FilterRule.or('one', 'two'),
        stringEquals: FilterRule.isEqual('test'),
        numericEquals: FilterRule.isEqual(1),
      }],
    });
```
I thought you meant it would look like this:
```
    new EventSourceMapping(stack, 'test', {
      target: fn,
      eventSourceArn: eventSourceArn,
      kafkaTopic: topicNameParam.valueAsString,
      filters: {
        orFilter: FilterRule.or('one', 'two'),
        stringEquals: FilterRule.isEqual('test'),
        numericEquals: FilterRule.isEqual(1),
      },
    });
```
who's contract would look like `{ [key: string]: any }`. In other words, I read too quickly and gave you a bad answer. I apologize for that. However, the other piece of this that I'd like to suggest an improvement upon is tightening the contract and not using `any` in here. See my comments in `FilterRule`.

Reply:
The problem is that the event source allows up to 5 filters (can be expanded if requested) and these filters can have several rules, so I put them in an array where each element would be a filter (with several rules)

Reply:
Ohhhhh. I see. Thanks for the extra explanation. No need to change this, then.

Reply:
I like this.

Reply:
Done